index.html

```diff
@@ -56,25 +87,24 @@ <h1 class="nombre"> Sergio<br> Valadez </h1>
       <img class="homeimg" src="images/landingdesk.png" alt="landingdesk" />
     </div>
   </section>
+  <!-- Land page end -->
+  <!-- Portfolio sections start here -->
   <section class="projects" id="projects">
     <h1 class="projects-title">Projects</h1>
-
     <img src="images/project1.png" width="750" class="i1" alt="project1">
     <div class="one projectcard">
       <div class="project-summary">
-
         <h2>Project's name goes here</h2>
         <ul class="languagebadge">
           <li class="language">HTML/CSS</li>
           <li class="language">Ruby on Rails</li>
           <li class="language">Javascript</li>
         </ul>
-        <button class="btn2" type="button">
+        <button class="btn2" id="openModal1" type="button">
           See this project
         </button>

```

Comment:
For every project, Kindly ensure that;
- [ ] when this button is clicked, the popup with details about the project appears.
- [ ] When the user clicks (or taps) the close (X) button, the popup disappears.
- [ ] The popup is responsive, it fits mobile and desktop sizes properly.
- [ ] The popup follows the Figma design that you have chosen.

Reply:
I'm working on it. Please keep an eye on the updates. It will be done before the deadline.